package com.prueba.certificacion.mispagos.exceptions;

public class GenerateExceptions extends AssertionError{

    public GenerateExceptions(String message, Throwable cause) {
        super(message, cause);
    }
}
